packages/flutter_tools/lib/src/device_categories.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
TBH I don't really see the value of pulling this out. device.dart is close to being unblocked, once the cache/artifacts/vmservice can land - I think we should wait

Reply:
Works for me.